packages/docusaurus/src/ssg/__tests__/ssgProgress.test.ts

```diff
@@ -0,0 +1,207 @@
+/**
```

Comment:
Unit tests do not pass

<img width="1160" height="524" alt="CleanShot 2025-08-24 at 20 25 14@2x" src="https://github.com/user-attachments/assets/390fd6e9-4997-4071-9f6d-62b47f1ad597" />


I haven't reviewed them yet but we need a CI that passes in any case